Source/displayinfo/include/ExtendedDisplayIdentification.h

```diff
@@ -199,8 +567,8 @@ namespace Plugin {
         // -------------------------------------------------------------
         // Accessors to the base information of the EDID raw buffer.
         // -------------------------------------------------------------
-        string Manufacturer() const {
-            string result;
+        std::string Manufacturer() const {
```

Comment:
Why this has changed?

Reply:
string is not a builtin or wpeframework type now, so must be namespaced